src/core/utils/Predicates.ts

```diff
@@ -0,0 +1,21 @@
+import { VNode } from '../stores/VNode';
+import { VNodeType } from '../stores/VNodeType';
+
+export type Predicate = (node: VNode) => boolean;
```

Comment:
Does it really need to be exported?

Reply:
It's used in VNode: https://github.com/odoo-dev/jabberwock/pull/41/files#diff-a8695298aff4077e84b4272d035925e1R3